tests/Nmap/Tests/NmapTest.php

```diff
@@ -258,7 +258,7 @@ public function testScanWithUserTimeout()
      */
     public function testExecutableNotExecutable()
     {
-        $executor = $this->getMock('Nmap\Util\ProcessExecutor');
+        $executor = $this->createMock('Nmap\Util\ProcessExecutor');
```

Comment:
If upgrading the PHP version to `php-5.6+`, the class instance string can use the `::class` magic method approach.